### PR TITLE
translate: complete the translated content

### PR DESCRIPTION
**Pull request by ikxin**

### CODE REVIEW

File: dashboard/locales/zh-TW/billing.json [hunk 1]
```diff
@@ -1,51 +1,51 @@
...
+        "title": "更新您的方案"
     },
     "plan": {
-      "current": "當前方案",
-      "upgrade": "升級",
-      "include": "它包括",
-      "free-credit": "五美金的免費額度",
-      "month": "（每月）",
-      "serverless": "你只能在這個方案上部署 serverless functions。"
+        "current": "當前方案",
```
Comment (pan93412) [inline dashboard/locales/zh-TW/billing.json:10]:
```suggestion
        "current": "目前方案",
```

File: dashboard/locales/zh-TW/billing.json [hunk 2]
```diff
@@ -1,51 +1,51 @@
...
-        "setup": "綁定"
-      },
-      "manage-section": {
-        "no-card": "沒有信用卡？",
-        "add": "添加",
-        "setup-alipay": "綁定支付寶",
-        "have-card": "有信用卡？"
-      }
+        "method": "付款方式",
+        "no-method": "你還沒有綁定付款方式。你將需要手動來打卡延長試用期限，以保證服務穩定運行。",
```
Comment (pan93412) [inline dashboard/locales/zh-TW/billing.json:19]:
```suggestion
        "no-method": "你還沒有綁定付款方式。你將需要手動打卡延長試用期限，以保證服務穩定運作。",
```

File: dashboard/locales/zh-TW/billing.json [hunk 3]
```diff
@@ -1,51 +1,51 @@
...
+        "card": "你綁定了一張信用卡，卡號結尾是",
+        "alipay": "你綁定了支付方式為",
```
Comment (pan93412) [inline dashboard/locales/zh-TW/billing.json:21]:
```suggestion
        "alipay": "你綁定了付款方式為",
```

File: dashboard/locales/zh-TW/project.json [hunk 4]
```diff
@@ -1,72 +1,72 @@
...
+        "cancel": "關閉",
+        "placeholder": "請輸入專案名稱"
+    },
+    "members": "成員",
+    "environments": "環境",
+    "services": "服務",
+    "add-env": "建立環境",
+    "add-service": "建立服務",
+    "settings": {
+        "title": "專案設置",
```
Comment (pan93412) [inline dashboard/locales/zh-TW/project.json:20]:
```suggestion
        "title": "專案設定",
```

File: dashboard/locales/zh-TW/project.json [hunk 5]
```diff
@@ -1,72 +1,72 @@
...
+        "general": {
+            "title": "通用設置",
```
Comment (pan93412) [inline dashboard/locales/zh-TW/project.json:22]:
```suggestion
            "title": "一般設定",
```

File: dashboard/locales/zh-TW/service.json [hunk 6]
```diff
@@ -24,7 +24,8 @@
         "history": "歷史紀錄",
         "toast": {
             "redeploy": "開始重新部署"
-        }
+        },
+        "no-domain": "為您的服務創建一個域名"
```
Comment (pan93412) [inline dashboard/locales/zh-TW/service.json:28]:
```suggestion
        "no-domain": "為您的服務建立一個域名"
```

File: dashboard/locales/zh-TW/service.json [hunk 7]
```diff
@@ -140,6 +138,10 @@
                 "title": "暫停服務",
                 "desc": "服務已暫停以避免額外收費。在您重新啟動之前，該服務將無法訪問。暫停超過30天的服務將自動刪除。"
             }
-        }
+        },
+        "wrong": "格式錯誤"
+    },
+    "networking": {
+        "title": "網絡連接"
```
Comment (pan93412) [inline dashboard/locales/zh-TW/service.json:145]:
```suggestion
        "title": "網路連線"
```

**Review by pan93412 — CHANGES_REQUESTED** [review]
Awesome works! I have only some suggestion for this.

**Comment by MichaelYuhe** [comment]
Great work, thanks for the help!

**Review by MichaelYuhe — APPROVED** [review]

**Review by pan93412 — APPROVED** [review]

**Pull request merged**